app/battle/creating_knights.py

```diff
@@ -0,0 +1,17 @@
+from app.knights.knights import Knight
+
+
+def creating_knight(name: str, knights_config: dict) -> Knight:
+    config = knights_config[name]
```

Comment:
Ensure that the `name` key exists in the `knights_config` dictionary to avoid potential `KeyError`. Consider using `knights_config.get(name)` to safely access the configuration.